engines/wintermute/base/file/base_disk_file.cpp

```diff
@@ -115,11 +115,24 @@ Common::SeekableReadStream *openDiskFile(const Common::String &filename) {
 
 	// Absolute path: TODO: Add specific fallbacks here.
 	if (fixedFilename.contains(':')) {
-		if (fixedFilename.hasPrefix("c:/windows/fonts/")) { // East Side Story refers to "c:\windows\fonts\framd.ttf"
-			fixedFilename = filename.c_str() + 14;
-		} else if (fixedFilename.hasPrefix("c:/carol6/svn/data/")) {	// Carol Reed 6: Black Circle refers to "c:\carol6\svn\data\sprites\system\help.png"
-			fixedFilename = fixedFilename.c_str() + 19;
-		} else {
+		// We try a few workarounds for known broken absolute paths in game scripts...
+		const char* const knownPrefixes[] = {
```

Comment:
On a second thought, I think it's slightly more readable right where it is.
I tried moving it and... I liked it slightly less.
If you strongly feel about the issue I don't object either way.

However, I added a // HACK comment to improve discoverability of... the hack.